src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,77 @@
         </div>
     {% endif %}
 
+    <h2>Security Firewall</h2>
+
+    {% if collector.firewall %}
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name }}</span>
+                <span class="label">Name</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.securityEnabled ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Security enabled</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.stateless ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Stateless</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.allowsAnonymous ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Allow Anonymous</span>
```

Comment:
I made myself the reflexion, but I wasn't sure it mattered as much in this context ^^. But for consistency, why not.